README.md

```diff
@@ -38,70 +38,95 @@ The second: you need not only a secure context for the browser, but actual safet
 
 ## Overview
 
-1. Get a server. It doesn't need to be big. Ideally you should have at least one slave, too, because NS entries require at least two servers.
-1. Point the NS entry of your domain to this server.
-1. [Install deps](#base-installation-and-deps).
-1. [Run dnsserver.py](#running-the-dns-server).
-1. [Create the certificates running certbotdns.py](#renewing-keys).
+1. Get an Ubuntu server. It doesn't need to be big. Ideally you should have at least one slave, too, because NS entries require at least two servers.
+2. Point the NS entry of your domain to this server.
+3. [Install deps](#base-installation-and-deps).
+4. [Run dnsserver.py](#running-the-dns-server).
+5. [Create the certificates running certbotdns.py](#renewing-keys).
+
+## Prerequisites 
+
+* Python 3.6 or above (see `util/ubuntu-install.bash` script )
+* certbot and the dnslib and cherrypy PIPs (see `util/ubuntu-install.bash` script )
+* Ubuntu 22.04 Server
+* Static IP
+* 4 DNS entrie for your TLD or sub-domain  (or sub sub domain etc.):  
+    * A record: `local-ip.medicmobile.org` -> `15.188.129.97`
```

Comment:
Why do you need this?

Reply:
Ah - do you mean the `A` record in general or the fact that I (kinda forgot I) used  `local-ip.medicmobile.org`?  

Either way, I think having those FQDNs and IPs which were specific to my org is weird, so I've fixed them in df483ce.

Otherwise, that `A` record is so that you can go to the website for the project, for example: https://local-ip.medicmobile.